src/components/Playroom/Playroom.tsx

```diff
@@ -69,6 +69,9 @@ export default () => {
     },
     dispatch,
   ] = useContext(StoreContext);
+
+  useDocumentTitle(title === undefined ? undefined : getTitle(title));
```

Comment:
`getTitle` isn't used elsewhere so it no-longer needs to accept `undefined` with this code.

But it then checks for truthy, which means it differs behaviour based only on "" (Line 40-47).

Is this intended? Otherwise can getTruthy be updated to just check for truthy/undefined and return undefined, and remove the `window?.__playroomConfig__.title` behaviour all together.

```suggestion
  useDocumentTitle(getTitle(title));
```

Reply:
Thoughts on just ditching `getTitle` all together?

```tsx
const configTitle = window?.__playroomConfig__?.title || 'Playroom';
const documentTitle = title ? `${title} | ${configTitle}` : configTitle;

useDocumentTitle(documentTitle);
```

Reply:
Tried to simplify another way. Let me know if you have feedback